Easydict/Swift/Feature/Localization/I18nHelper.swift

```diff
@@ -25,12 +25,20 @@ class I18nHelper: NSObject {
     }
 
     var localizeCode: String {
-        UserDefaults.standard.string(forKey: languagePreferenceLocalKey) ?? LanguageState.LanguageType
-            .simplifiedChinese.rawValue
+        guard let code = UserDefaults.standard.string(forKey: languagePreferenceLocalKey) else {
```

Comment:
Looks like 'if let' is more suit for this case here.

FYI： https://www.hackingwithswift.com/quick-start/understanding-swift/when-to-use-guard-let-rather-than-if-let